src/config.rs

```diff
@@ -30,28 +38,93 @@ use crate::error::Result;
 use serde::{Deserialize, Serialize};
 use std::path::Path;
 
+/// Validates PostgreSQL schema name according to SQL identifier rules
+///
+/// Rules from PostgreSQL documentation:
+/// - Must begin with a letter (a-z, A-Z) or underscore (_)
+/// - Subsequent characters can be letters, underscores, digits (0-9), or dollar signs ($)
+/// - Maximum length is 63 bytes (NAMEDATALEN-1)
+///
+/// # Arguments
+/// * `schema` - The schema name to validate
+///
+/// # Returns
+/// * `Ok(())` if the schema name is valid
+/// * `Err(PgqrsError::InvalidConfig)` if the schema name is invalid
+pub fn validate_schema_name(schema: &str) -> Result<()> {
```

Comment:
Rename to `validate_identifier`. This function will be used to verify queue names as well.